OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
+
+                    if (useDistortion)
                     {
                         //@todo figure out which type of distortion to use
                         //right now, there is only one option, SurfaceRadialDistortion
                         //get distortion parameters
-                        OSVR.ClientKit.RadialDistortionParameters distortionParameters =
-                        Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(
-                        Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
+                        OSVR.ClientKit.RadialDistortionParameters distortionParameters;
+
+                        try
+                        {
+                            distortionParameters = Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
```

Comment:
OSVR-Monogame gets those values from the json descriptor which works well. I think we must do the same thing.
I agree for the HDK2, maybe the renderManager port of Managed OSVR can solve all these issues?  Pretty confident with that.